### PR TITLE
Add option to set registration marks distance from components

### PR DESCRIPTION
**Pull request by ernierasta**
Hi,
solves #65.
I was not sure how to call option to set distance of cutting (registration) marks ...
Will happily correct whatever is needed. :-)

### CODE REVIEW

**Comment by lifelike** [comment]
Looks good, except you forgot to update the margins calculation around line 1400 or so to include this extra space. That has to be updated or the components plus registration lines will not fit inside of the page when there is no cs_layout layer.

Happy to merge once that is fixed. :)

**Comment by ernierasta** [comment]
Thank you for looking at it. Fixed.

File: countersheet.py [hunk 1]
```diff
@@ -1402,7 +1402,7 @@ def effect(self):
                                    docwidth,
                                    self.getViewBoxHeight(svg))]
             margin = max(self.registrationmarkslen,
-                         self.outlinedist)
+                         self.outlinedist) + self.registrationmarksdist
```
Comment (lifelike) [inline countersheet.py:1405]: I think it should only be added to registrationmarkslen (in the call to max), not added like that?

Comment (ernierasta) [reply countersheet.py]: Done. :-)

**Comment by ernierasta** [comment]
I am sure you know how to solve this conflict(that is a pity there is no mechanism I am aware of, to propose conflict resolving), but it should by like that:
```python
            positions = [Rectangle(0.0,
                                   0.0,
                                   docwidth,
                                   docheight)]
            margin = max(self.registrationmarkslen + self.registrationmarksdist,
                         self.outlinedist)
```

**Comment by lifelike** [comment]
No worries. I solved the conflicts last night, just got interrupted before pushing it back to github.

**Pull request merged**